appinventor/components/build.xml

```diff
@@ -152,6 +152,7 @@
     <copy toFile="${public.deps.dir}/commons-pool.jar" file="${lib.dir}/commons-pool/commons-pool2-2.0.jar" />
     <copy toFile="${public.deps.dir}/guava-14.0.1.jar" file="${lib.dir}/guava/guava-14.0.1.jar" />
     <copy toFile="${public.deps.dir}/core.jar" file="${lib.dir}/QRGenerator/core.jar" />
+    <copy toFile="${public.deps.dir}/physicaloid.jar" file="${lib.dir}/physicaloid/physicaloid-library.jar" />
```

Comment:
What's the min SDK needed for this lib? If it's not minSdk 7, please specify in the component.

Reply:
Sorry, totally forgot about it. As we have a higher min sdk I didn't add it here. It is Android 3.1 = API 12.